lib/pbench/server/api/resources/query_apis/datasets/namespace_and_rows.py

```diff
@@ -209,7 +209,7 @@ def assemble(self, json_data: JSON, context: CONTEXT) -> JSON:
         from a selected set of documents that belong to the given run id in
         the specified index.
```

Comment:
Here's an off-hand comment I'd considered earlier but rejected since we were following the guideline of supporting the current dashboard model... I was never entirely comfortable with changing our access criteria from "controller"/"name" at the "higher" levels to "run_id" at these "lower" levels. But that's how the dashboard worked. I also didn't really want to spread the "controller" out to more APIS.

But now, especially since I'm in the process of removing "controller" from the APIs ... I like the consistency of moving towards using dataset name for identification of a dataset, throughout. (And if that's a bit more work for the current dashboard code, "so what"? 🤷🏻 )

Reply:
Right using the dataset name instead of run id to identify the dataset is something we would want to do anyway.